src/pages/HomePage/HomePage.tsx

```diff
@@ -21,10 +21,12 @@ import { motion } from 'framer-motion';
 const HomePage = () => {
   return (
     <Grid gap={8}>
+	<Flex justifyContent="center">	
       <Link
         _hover={{ textDecoration: 'none' }}
         href="https://github.com/OpenSource-Journey/Your-Github-Contributions"
         target="_blank"
+		style={{width:'max-content'}}
```

Comment:
@Hat52 , Chakra UI supports CSS in JSX, so you don't have to use the style object to give the styling to the element.

For more you can refer this https://chakra-ui.com/getting-started
```suggestion
		width:'max-content'
                 margin='auto'
```